docs/current_docs/api/arguments.mdx

```diff
@@ -458,66 +457,66 @@ The best way to understand this is with an example. Consider the following direc
+| `..` | `/` | `dagger call -m my-module read-dir` | `Error: path outside context directory` |
+| `/README.md` | `/` | `dagger call -m my-module read-file` | `/README.md` |
+| `./README.md` | `/` | `dagger call -m my-module read-file` | `/README.md` |
+| `../README.md`| `/`  | `dagger call -m my-module read-file` | `Error: path outside context directory` |
 
 For a non-Git directory:
 
-| Default context ('defaultPath`) | Dagger Function invocation | `source` resolves to |
-| --- | --- | --- |
-| `/` | `dagger call -m my-module read-dir` | `/my-module` |
-| `/src` | `dagger call -m my-module read-dir` | `/my-module/src` |
-| `.` | `dagger call -m my-module read-dir` | `/my-module` |
-| `..` | `dagger call -m my-module read-dir` | Error: escaping default context directory root is not allowed |
-| `/README.md` | `dagger call -m my-module read-file` | Error: file does not exist |
-| `./README.md` | `dagger call -m my-module read-file` | Error: file does not exist |
-| `../README.md` | `dagger call -m my-module read-file` | Error: escaping default context directory root is not allowed |
+| Default path ('defaultPath`) | Context directory | Dagger Function invocation | `source` argument resolves to |
+| --- | --- | --- | --- |
+| `/` | `/my-module` | `dagger call -m my-module read-dir` | `/my-module` |
+| `/src` | `/my-module` | `dagger call -m my-module read-dir` | `/my-module/src` |
+| `.` | `/my-module` | `dagger call -m my-module read-dir` | `/my-module` |
+| `..` | `/my-module` | `dagger call -m my-module read-dir` | `Error: path outside context directory` |
+| `/README.md` | `/my-module` | `dagger call -m my-module read-file` | `Error: file does not exist` |
+| `./README.md` | `/my-module` | `dagger call -m my-module read-file` | `Error: file does not exist` |
+| `../README.md` | `/my-module` | `dagger call -m my-module read-file` | `Error: path outside context directory` |
 
-If a value is explicitly passed to the Dagger Function argument, it always overrides the default context.
+If a value is explicitly passed to the Dagger Function argument, it always overrides the default path.
 
-| Default context ('defaultPath`) | Dagger Function invocation | `source` resolves to |
+| Default path ('defaultPath`) | Dagger Function invocation | `source` argument resolves to |
 | --- | --- | --- |
-| `/` | `dagger call -m my-module read-dir --source=public` | `/public` |
-| `/src` | `dagger call -m my-module read-dir --source=public` | `/public` |
-| `.` | `dagger call -m my-module read-dir --source=public` | `/public` |
-| `..` | `dagger call -m my-module read-dir --source=public` | `/public` |
+| `/` | `dagger call -m my-module read-dir --source=public` | `./public` |
+| `/src` | `dagger call -m my-module read-dir --source=public` | `./public` |
+| `.` | `dagger call -m my-module read-dir --source=public` | `./public` |
+| `..` | `dagger call -m my-module read-dir --source=public` | `./public` |
 
 :::tip
```

Comment:
You should link that to https://github.com/dagger/dagger/pull/8739

Reply:
Done